scripts/globals/abilities/pets/mountain_buster.lua

```diff
@@ -23,5 +23,10 @@ function onPetAbility(target, pet, skill)
 	target:delHP(totaldamage);
 	target:updateEnmityFromDamage(pet,totaldamage);
 
+	if (damage > 0) then
+	    if (target:hasStatusEffect(EFFECT_BIND) == false) then
+	    target:addStatusEffect(EFFECT_BIND, 1, 0, duration);

```

Comment:
You need to define duration somewhere before this line, or just use a number directly. If duration isn't holding a number, this would crash a server upon use.